src/app/admin/page.tsx

```diff
@@ -1,5 +1,235 @@
-const page = () => {
-  return <div>ADMIN DASHBOARD</div>;
+import DeleteButton from "@/components/DeleteButton";
+import BasicLineChart from "@/components/LineChart";
+import {
+  countOrders,
+  lastThirtyDaysOrders,
+  lastThirtyDaysOrdersNotPaid,
+  lastThirtyDaysOrdersPaid,
+  lastThirtyDaysSales,
+  topFiveProductsLastThirtyDays,
+} from "@/lib/utils/orders";
+import { countProducts } from "@/lib/utils/products";
+import { countUsers, lastThirtyDaysUsers } from "@/lib/utils/users";
+
+const page = async () => {
+  const userChartData = await lastThirtyDaysUsers();
+  const paidOrders30d = await lastThirtyDaysOrdersPaid();
+  const notPaidOrders30d = await lastThirtyDaysOrdersNotPaid();
+  const orders30d = await lastThirtyDaysOrders();
+  const totalSales30d = await lastThirtyDaysSales();
+
+  const topFiveProd = await topFiveProductsLastThirtyDays();
+
+  const userCount = await countUsers();
+  const productCount = await countProducts();
+  const orderCount = await countOrders();
+
+  return (
+    <div
+      id="src-app-admin-page-mainContainer"
+      className="flex flex-col w-full h-full overflow-y-auto"
+    >
+      <div
+        id="src-app-admin-page-mainContainer-gridContainer"
+        className="w-full p-4 gap-5 items-center grid grid-cols-1 grid-rows-3 md:grid-cols-2 md:grid-rows-2 xl:grid-cols-3 xl:grid-rows-1"
+      >
+        <div
+          id="src-app-admin-page-mainContainer-gridContainer-newUsers30d"
+          className="flex flex-col gap:1 justify-start rounded-xl items-center bg-white shadow-xl"
+        >
+          <h1
+            id="src-app-admin-page-mainContainer-gridContainer-newUsers30d-title"
+            className="text-2xl font-bold mt-2"
+          >
+            New users 30D
+          </h1>
+          <BasicLineChart
+            xAxis={[{ data: userChartData.map((entry) => entry.dayNumber) }]}
```

Comment:
This is producing wrong results, instead of using this value we should use the real date

```suggestion
            xAxis={[
              {
                data: userChartData.map((entry) => entry.date),
                scaleType: "time",
              },
            ]}
```